plugins/module_utils/mh/base.py

```diff
@@ -63,3 +63,10 @@ def run(self):
         if 'failed' not in output:
             output['failed'] = False
         self.module.exit_json(changed=self.has_changed(), **output)
+
+    @classmethod
+    def execute(cls, module=None):
+        instance = cls(module)
+        instance.run()
```

Comment:
Nit: an intermediate var is probably unnecessary, is it?
```suggestion
        cls(module).run()
```